subprojects/ludocode-mpack.wrap

```diff
@@ -1,7 +1,7 @@
 [wrap-file]
-directory = mpack-1.0
-source_url = https://github.com/ludocode/mpack/archive/v1.0.tar.gz
-source_filename = ludocode-mpack-1.0.tar.gz
-source_hash = f80fc0539554dc58eaa22f6260e91709366475b01b2d004372e98a59d08388ac
+directory = mpack-1.1
+source_url = https://github.com/ludocode/mpack/archive/v1.1.tar.gz
+source_filename = ludocode-mpack-1.1.tar.gz
+source_hash = a62a3954dbbd77cf7f3048e8d7e89718d45a851d55003c37c51277c71ca968e6
 patch_directory = ludocode-mpack
```

Comment:
You'll need to add a `[provide]` entry to this wrap. It was not a requirement when the original was created, but the the rules have changed since then.